index.html

```diff
@@ -73,8 +73,99 @@
       </div>
     </div>
   </header>
-  <main></main>
+
+    <main>
+     <section class="product">
+      <div class="carousel">
+      <div class="product-carousel">
+        <div class="carousel-img">
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+          <div class="carousel-item product-banner" style="background-image: url('image/Banner.jpg')"></div>
+        </div>
+      <div class="indicators">
+        <span class="dot" onclick="currentSlide(0)"></span>
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
```

Comment:
- Los span no son elementos interactivos
- Separa lo mas que puedas la lógica de la vista. Puedes usar la estrategia con clases `js-` que solo se usen para las acciones que tengan que ver con javascript
```suggestion
        <button type="button" class="dot js-go-to-slide" data-slide="0"></button>
        <button type="button" class="dot js-go-to-slide" data-slide="1"></button>
        <button type="button" class="dot js-go-to-slide" data-slide="2"></button>
```

Y en el js algo asi

```
const slideButtons = document.querySelectorAll('.js-go-to-slide');
slideButtons.foreach(slideButton => {
  slideButton.addEventListener('click', { target } => currentSlide(parseInt(target.dataset.slide)));
});
```